CollapsibleExample/MHCollapsibleSection.m

```diff
@@ -22,6 +25,7 @@ - (MHFilterLabel*)returnOffsetLabel:(NSUInteger)row;
 
 @implementation MHCollapsibleSection
 
+#pragma Initialize Section
 - (instancetype) initWithArray:(NSArray*)filters headerTitle:(NSString*)headerTitle
                      animation:(UITableViewRowAnimation)animation rowRange:(NSRange)rowRange{
     self = [self init];
```

Comment:
You should check if init succeeded or failed before doing your additional initialization ie
```
self = [self init];
if (self) {
...
}
return self;
```